gamma/2.1/Dockerfile

```diff
@@ -0,0 +1,107 @@
+# base image
+FROM ubuntu:bionic as app
+
+# for easy upgrade later. ARG variables only persist during image build time
+ARG GAMMA_VER="2.1"
+
+# metadata
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="2"
+LABEL software="centroid"
+LABEL software.version="v2.1"
+LABEL description="Gene Allele Mutation Microbial Assessment"
+LABEL website="https://github.com/rastanton/GAMMA/"
+LABEL license="Apache 2.0"
+LABEL maintainer="Jill Hagey"
+LABEL maintainer.email="jvhagey@gmail.com"
+
+# install dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    pigz \
+    rsync \
+    unzip \
+    tar \
+    python3.7 \
+    python3-pip \
+    gzip \
+    wget \
+    git \
+    curl \
+    make \
+    g++ \
+    ncbi-blast+ \
+    hmmer && \
+    apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/*
+
+#install biopython
+RUN pip3 install biopython cgecore tabulate
+
+# hmmer 3.3 since amrfinder requires 3.2 or higher (apt version is 3.1)
+# apt dependencies: make, g++
+RUN wget http://eddylab.org/software/hmmer/hmmer-3.3.tar.gz && \
+ tar xzf hmmer-3.3.tar.gz && \
+ rm hmmer-3.3.tar.gz && \
+ cd hmmer-3.3 && \
+ ./configure && \
+ make && \
+ make install
+
+# install amrfinder
+RUN mkdir amrfinder && cd /amrfinder && \
+    wget https://github.com/ncbi/amr/releases/download/amrfinder_v3.10.18/amrfinder_binaries_v3.10.18.tar.gz && \
+    tar zxf amrfinder_binaries_v3.10.18.tar.gz && \
+    rm amrfinder_binaries_v3.10.18.tar.gz
+
+# install blat
+RUN wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/blat/blat && \
+    chmod +x blat && \
+    mv blat /bin/blat && \
+    mkdir /data
+
+# download ARG-ANNOT database (https://github.com/rastanton/GAMMA/issues/6)
+RUN wget http://backup.mediterranee-infection.com/arkotheque/client/ihumed/_depot_arko/articles/691/argannot-nt_doc.fasta && \
+ mv -v argannot-nt_doc.fasta arg-annot.fasta

```

Comment:
There does not appear to indicate a version for this either. The last time this was updated was 2015-12-15